### PR TITLE
Removed optiGraphicbufferupload flag

### PR DESCRIPTION
**Pull request by emilwestergren**
This is not the place for it.
@marcusnaslund peer review

### CODE REVIEW

File: source/draw/gpu/opengl/AndroidContext.ooc [hunk 1]
```diff
@@ -27,19 +27,14 @@ AndroidContext: class extends OpenGLContext {
...
 				rgba referenceCount decrease()
-			}
-			else
-				result = super(rasterImage)
-		} else
-			result = super(rasterImage)
-		result
+				result
+			case => super(rasterImage)
+		}
```
Comment (marcusnaslund) [inline source/draw/gpu/opengl/AndroidContext.ooc]: This method isn't "simple" enough to warrant returning in two places.
`result := match (rasterImage) { ...`
and then
`result`
will suffice.

Comment (emilwestergren) [inline source/draw/gpu/opengl/AndroidContext.ooc]: It doesn't return in two places, `match` always returns a single value.

Comment (marcusnaslund) [inline source/draw/gpu/opengl/AndroidContext.ooc]: Yes, but depending on what you are matching, you may either return the `result` variable declared on line 33 or the result from the super function on line 36.

Comment (marcusnaslund) [inline source/draw/gpu/opengl/AndroidContext.ooc]: The `FloatTuple` covers for example return using a `match` statement like this. The difference is that each case statement there is a single line, so it's easy to follow.

Comment (emilwestergren) [inline source/draw/gpu/opengl/AndroidContext.ooc]: Why not simply always allow this? I don't think it improves readability by adding a `result` variable to be fair.

Comment (marcusnaslund) [inline source/draw/gpu/opengl/AndroidContext.ooc]: What about if other `case`s are introduced? Or if other code is added besides the `match`?
Enforcing this rule does not add anything in *this* particular case, I agree, but it does future-proof the code. And helps consistency.

Comment (marcusnaslund) [inline source/draw/gpu/opengl/AndroidContext.ooc]: Actually, I think I'll withdraw my position on this. The futureproof argunent doesn't really hold.

Comment (emilwestergren) [inline source/draw/gpu/opengl/AndroidContext.ooc]: Your call, I've voiced my opinion at least :)

Comment (marcusnaslund) [inline source/draw/gpu/opengl/AndroidContext.ooc]: And it's been processed. Thank's for the discussion, we will go with this.

**Pull request merged**